keps/sig-apps/3998-job-success-completion-policy/README.md

```diff
@@ -759,9 +615,11 @@ No.
 
 ###### How can an operator determine if the feature is in use by workloads?
 
-`job_finished_total`: the new `JobSuccessPolicy` and `Completions` values for the `reason` label. 
-  - `JobSuccessPolicy` indicates a job is declared as succeeded because the job meets `.spec.successPolicy`.
-  - `Completions` indicates a job is declared as succeeded because the job meets `spec.completions`.
+We will introduce the new `job_succeeded_total` metric with `JobSuccessPolicy` and `Completions` reasons,
+which indicates the following situations: 
+
+- `JobSuccessPolicy` indicates a job is declared as `SuccessCriteriaMet` because the job meets `spec.succesPolicy`.
+- `Completions` indicates a job is declared as `SuccessCriteriaMet` because the job meets `spec.completions`.
```

Comment:
> Will enabling / using this feature result in any new API calls?

Please remove the "No" from there :)

Reply:
Oops, I was supposed to remove the "No" in the alpha stage :)

Thank you!